modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: PROCEDURE
+[id="sno-adding-worker-nodes-to-single-node-clusters-manually_{context}"]
+= Adding worker nodes to {sno} clusters manually
+
+You can add a worker node to a {sno} cluster manually by booting the worker node from {op-system-first} ISO and using the cluster `worker.ign` file to join the new worker node to the cluster.
+
+.Prerequisites
+
+* Install a {sno} cluster on bare metal.
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Ensure that all the required DNS records exist for the cluster that you are adding the worker node to.
+
+.Procedure
+
+. Set the {product-title} version:
++
+[source,terminal]
+----
+$ OCP_VERSION=<ocp_version> <1>
+----
++
+<1> Replace `<ocp_version>` with the current version, for example, `latest-{product-version}`
+
+. Set the host architecture:
++
+[source,terminal]
+----
+$ ARCH=<architecture> <1>
+----
+<1> Replace `<architecture>` with the target host architecture, for example, `aarch64` or `x86_64`.
+
+. Get the `worker.ign` data from the running single-node cluster by running the following command:
++
+[source,terminal]
+----
+$ oc extract -n openshift-machine-api secret/worker-user-data --keys=userData --to=- > worker.ign
+----
+
+. Host the `worker.ign` file on a web server accessible from your network.
+
+. Download the {product-title} installer and make it available for use by running the following commands:
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OCP_VERSION/openshift-install-linux.tar.gz > openshift-install-linux.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxvf openshift-install-linux.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x openshift-install
+----
+
+. Retrieve the {op-system} ISO URL:
++
+[source,terminal]
+----
+$ ISO_URL=$(./openshift-install coreos print-stream-json | grep location | grep $ARCH | grep iso | cut -d\" -f4)
+----
+
+. Use the RHCOS ISO and the hosted `worker.ign` file to install the worker node:
```

Comment:
You should expand RHCOS on first use, and {op-system-first} is the attribute you want for it.